### PR TITLE
Reports: Copy updates

### PR DESCRIPTION
**Pull request by claudiovallejo**
**Story card:** https://app.clubhouse.io/simpledotorg/epic/48/dashboard-reports-v1

## Because

Improving copy per Daniel/Reena's feedback prior to initial release

## This addresses

**"No recent BP"**
*Label copy => "No visit >3 months" and "Visited in the last 3 months"*
image

**"Cumulative registrations"**
*Update tooltip to read "XX cumulative registrations in XX"*
image

**Cards 1 - 3, "Visit details"**
*Add info tooltip with “Patients registered until the end of April 2020” to Cards 1 - 3 and "Visit details..." card*
image

**Denominator tooltips**
*Update cards 1 - 3 and "Visit details" to include denominator. Example "5% control rate (102 of XX patients) in July 2020"*
image

**"Visit details of all HTN patients"**
*Added subtitle; Made some minor spacing polishes and cleaned up HTML*
image

**Updated 'Early access' banner copy**
*Added list of upcoming features*
image

### CODE REVIEW

File: app/views/reports/regions/show.html.erb [hunk 1]
```diff
@@ -116,9 +124,15 @@
...
             <% if @data.last_value(:uncontrolled_patients_rate) %>
-              <span class="fw-bold">
-                <%= number_with_delimiter(@data.last_value(:uncontrolled_patients)) %>
-                </span> of <%= number_with_delimiter(@data.last_value(:adjusted_registrations)) %> patients
+              <span class="fw-bold"><%= number_with_delimiter(@data.last_value(:uncontrolled_patients)) %></span> of <%= number_with_delimiter(@data.last_value(:adjusted_registrations)) %> patients
+              <span
+                data-toggle="tooltip"
+                data-placement="top"
+                data-trigger="hover"
+                title="Patients registered until the end of <%= @data[:adjusted_registrations].keys[-4] %>"
```
Comment (rsanheim) [inline app/views/reports/regions/show.html.erb:132]: Can you extract this to an instance variable in the controller, and then use that in the views?  Grabbing it with`-4` everytime is a little awkward.

File: app/views/reports/regions/show.html.erb [hunk 2]
```diff
@@ -74,12 +82,12 @@
...
               </p>
               <p class="flex-1 m-0px fs-14px">
-                Visited in the last 3 months but no BP measure
+                Visited in the last 3 months
                 <span
                   data-toggle="tooltip"
                   data-placement="top"
                   data-trigger="hover"
-                  title="Patients with an appointment scheduled, updated medicines, or blood sugar taken"
+                  title="Patients registered until the end of <%= @data[:adjusted_registrations].keys[-4] %> with an appointment scheduled, updated medicines, or blood sugar taken"
```
Comment (rsanheim) [inline app/views/reports/regions/show.html.erb:90]: Now that I'm reading this, we maybe shouldn't put the month here. It implies we are going back 3 three months from today for the _entire_ graph, but its going back 3 months for each month...so its a relative measure.  Maybe just clarify that in the titles?

Comment (rsanheim) [reply app/views/reports/regions/show.html.erb:90]: image

The position of the tooltip implies it applies to the whole graph here...

Comment (claudiovallejo) [reply app/views/reports/regions/show.html.erb:90]: Great point @rsanheim! Updated it to read "Patients registered 3 months prior with an appointment scheduled..."

File: app/controllers/reports/regions_controller.rb [hunk 3]
```diff
@@ -22,6 +22,7 @@ def show
     @quarterly_registrations = @data[:quarterly_registrations]
     @last_registration_value = @data[:cumulative_registrations].values&.last || 0
     @new_registrations = @last_registration_value - @data[:cumulative_registrations].values[-2]
+    @adjusted_registration_date = @data[:adjusted_registrations].keys[-4]
```
Comment (rsanheim) [inline app/controllers/reports/regions_controller.rb:25]: think you'll need this in the details action as well

Comment (claudiovallejo) [reply app/controllers/reports/regions_controller.rb:25]: Thanks!

**Pull request merged**